### PR TITLE
BZ1994413: Adding info about firewalld to NOTE

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.6+
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1994413
Preview Link: [Updating RHEL compute machines in your cluster](https://deploy-preview-35866--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-rhel-compute?utm_source=github&utm_campaign=bot_dp#rhel-compute-updating_updating-cluster-rhel-compute) See Step 1 NOTE.
QE ack required.

### CODE REVIEW

**Review by EricPonvelle — APPROVED** [review]
/lgtm

**Comment by xltian** [comment]
@gpei to review from QE.

File: modules/rhel-compute-updating.adoc [hunk 1]
```diff
@@ -34,7 +34,7 @@ your RHEL machines and the `upgrade` playbook.
 +
 [NOTE]
 ====
-You must not enable firewalld later. If you do, you cannot access {product-title} logs on the worker.
+By default, the firewalld service is enabled on the host during installation. Having the firewalld service enabled on your host prevents you from accessing {product-title} logs on the worker. Do not enable firewalld later if you wish to continue accessing {product-title} logs on the worker.
```
Comment (gpei) [inline modules/rhel-compute-updating.adoc:37]: The firewalld service would be disabled during RHEL scale-up - https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/tasks/config.yml#L22.
So "By default, the firewalld service is enabled on the host during installation" is not correct.

Comment (sagidlow) [reply modules/rhel-compute-updating.adoc:37]: @gpei - Is firewalld enabled by default at any point? If not, then I can drop that statement.

Comment (gpei) [reply modules/rhel-compute-updating.adoc:37]: According to https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/tasks/config.yml#L19-L20 
`The base OS RHEL with "Minimal" installation option is enabled firewalld serivce by default`

**Comment by sagidlow** [comment]
@gpei Made the update. Let me know if that looks okay.

**Comment by gpei** [comment]
Thanks for the update, lgtm now

**Pull request merged**